lms/static/js/courseware/base/views/proctored_exam_view.js

```diff
@@ -0,0 +1,42 @@
+;(function (define) {
+
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'gettext'
+], function ($, _, Backbone, gettext) {
+
+   'use strict';
+
+    return Backbone.View.extend({
+        initialize: function (options) {
+            this.$el = options.el;
+            this.model = options.model;
+            this.templateId = options.proctored_template;
+            this.template = _.template($(this.templateId).html());
+            this.timerId = null;
+        },
+
+        render: function () {
+            var html = this.template(this.model.toJSON());
```

Comment:
Can we put in a null-guard on this.template? This view might get added to a page which does not have the underscore template rendered on it and I think as it is right now will cause a JS exception. That might explain why there were some acceptance test failures. In any case, seems like we should make sure that the template exists...